types/chai-style/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Type definitions for chai-style 1.0
+// Project: https://github.com/darlanmendonca/chai-style
+// Definitions by: Avi Vahl <https://github.com/AviVahl>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 3.0
```

Comment:
I think you can remove this line as there's nothing using version 3.0

Reply:
`@types/chai` themselves requires ts 3.0+:
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/chai/index.d.ts

so I guessed any assertion plugin (that imports chai) also has that requirement. Is that not correct?

Reply:
Aha, makes sense. Thanks for checking.